src/server/server.ts

```diff
@@ -616,6 +616,19 @@ namespace ts.server {
         return logEnv;
     }
 
+    function getLogPath(args: string[], initialIndex: number) {
+        let pathStart = args[initialIndex + 1];

```

Comment:
Why would you pass this function the index of the arg before (needing the +1 here)? Why not pass i + 1 in the call to getLogPath?

Reply:
I just thought it was cleaner this way. No strong opinions about either direction.

Reply:
And API should be intuitive and reusable without understanding the implementation details where possible. Having the path parts start _one after_ the index passed is not very intuitive.

Reply:
Fixed.